### ISSUE TITLE
Outdated benchmark (2y stale) mentioned in README

### ISSUE THREAD

**Issue by corneliusroemer** [pola-rs/polars#6840]
### Research

- [X] I have searched the above polars tags on Stack Overflow for similar questions.

- [ ] I have asked my usage related question on [Stack Overflow](https://stackoverflow.com).

### Question about Polars

In the README you prominently refer to the H2O database benchmark: https://github.com/h2oai/db-benchmark which is unfortunately no longer maintained for almost 2 years now - so it may be worth looking for an alternative and add a disclaimer that the benchmark is out of date. Sure, pandas won't have changed much but maybe some of the other tools.

You refer to it here: https://github.com/pola-rs/polars/blob/f81ba2f63890048e783c5a47e5a8b4ee40529e77/README.md?plain=1#L103

That line was edited before the benchmark became unmaintained. Probably worth considering how to move on.

**Comment by ritchie46** [comment]
We also show our own benchmarks. That is done recently. The h20ai has a lot of tools which makes it a lot of work for us to replicate.

**Comment by corneliusroemer** [comment]
Yes, absolutely, I can imagine that you don't want to replicate those benchmarks. Maybe just add the date of the last benchmark in parentheses - it isn't obvious that the benchmarks are outdated when one goes to the website unfortunately as it says it's continuously updated. Well, until it wasn't anymore...

Only when you go into the details do you notice it's 2y stale. Not meant as a criticism, just for transparency!

**Comment by deanm0000** [comment]
The benchmark's on the polars website link [here](https://www.pola.rs/benchmarks.html) which are newer.

The link from the manual doesn't tout any of the h20 results so to see the results it to also see that they're stale so to disclaim their age seems redundant.  

I don't mean to be too argumentative but how is it not obvious that the results are old, the dates are right there in the png

![image](https://user-images.githubusercontent.com/37878412/218523791-aca42082-1fec-49cb-b287-bc1cc131b047.png)

**Comment by corneliusroemer** [comment]
Yeah you're right, I somehow forgot about that. Though if you read the second sentence says "It runs regularly against very latest versions of these packages nd automatically updates". So at least that part is wrong.

Nice that there are more up to date benchmarks on the website! Maybe link to them as well? Could be a good update.

**Comment by stinodego** [comment]
This is no longer the case, as we now link to DuckDB's updated benchmarks, as well as our own TPCH benchmarks.
